firebase_admin/_auth_utils.py

```diff
@@ -275,6 +277,135 @@ def validate_provider_ids(provider_ids, required=False):
         validate_provider_id(provider_id, True)
     return provider_ids
 
+
+def validate_mfa_config(mfa_config):
```

Comment:
This method is way too long, please consider separate each if block to a single method.  This way it increases the readability.